src/System.Memory/src/System/Number/Number.FormatAndParse.cs

```diff
@@ -3,9 +3,10 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
This and the files changed bellow are wrong, we don't want to use this implementation at all. We want to use the implementation located in `src/CoreLib/System/Number` as a replacement for existing referencesource code

Reply:
This implementation is another "part" of the `partial class Number`.  Long term, we want to switch the other part over to the corefx version.  Once that's done, we still need these parts here - but they should be moved back inside the `partial class Number`.